src/actions/get_all_events.js

```diff
@@ -0,0 +1,25 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+const getAllEvents = (hbs, response) => {
+  mockEvents.find({}, (error, events) => {
+    if (error) response.render('not_found')
+    else {
+      let upcomingEvents = [];
+      let pastEvents = [];
+      events.forEach((event) => {
+        if(event.eventDate > new Date()) {
```

Comment:
Javascript has no problem with dates in if statement and its working, and I think event.eventDate returns the same date eyntax.
And for the date format we have a helper function to change it to String when its rendered to the page.